src/System.Collections.Specialized/src/System.Collections.Specialized.builds

```diff
@@ -4,8 +4,14 @@
   <ItemGroup>
     <Project Include="System.Collections.Specialized.csproj" />
     <Project Include="System.Collections.Specialized.csproj">
+      <TargetGroup>netstandard1.3</TargetGroup>
+    </Project>
+    <Project Include="System.Collections.Specialized.csproj">
       <TargetGroup>net46</TargetGroup>
```

Comment:
(Would it make sense to do such cleanup after this PR?  We have a bunch of other projects that would need to similarly be modified.)